docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -2,66 +2,10 @@ import { Alert, Tabs, TabItem } from '@aws-amplify/ui-react';
 import { TerminalCommand } from '@/components/InstallScripts';
 import { Fragment } from '@/components/Fragment';
 
-## Webpack
-
-### Webpack 5+: `Uncaught ReferenceError: process is not defined`
-
-Follow the instructions below to if you are using Webpack 5:
-
-<Alert role="none" variation="info">
-*Note:*
-Polyfill is only required for Webpack 5+. In this version Node.js global variable shims required by the `aws-amplify` package were removed, which results in the following error message:
-
-```
-Uncaught ReferenceError: process is not defined
-```
-
-Webpack 4 already includes this polyfill.
-
-</Alert>
-
-1. Add
-   [node-polyfill-webpack-plugin](https://www.npmjs.com/package/node-polyfill-webpack-plugin)
-   as dev dependency:
-
-<Tabs>
-<TabItem title="npm">
-<TerminalCommand command="npm install node-polyfill-webpack-plugin -D" />
-</TabItem>
-<TabItem title="yarn">
-<TerminalCommand command="yarn add node-polyfill-webpack-plugin -D" />
-</TabItem>
-</Tabs>
-
-2. Add the plugin to your `webpack.config.js` plugins list:\_
-
-```js
-  plugins: [
-    new NodePolyfillPlugin(), // Polyfill Node.js globals (e.g. global, process, etc)
-  ],
-```
-
-### Webpack 4: Can’t import the named export ‘Amplify’ from non EcmaScript module (only default export is available)
-
-Follow [here](#cra-4-cant-import-the-named-export-amplify-from-non-ecmascript-module-only-default-export-is-available) for solutions.
-
-## Next.js
-
-### Next 13.4+: React Server Components
-
-Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
-
-## Vite
-
-When working with a [Vite](https://vitejs.dev) project you must make a few modifications. Please follow the steps below.
-
-<Fragment>
-  {({ platform }) => import(`./shared/vite-polyfill.mdx`)}
-</Fragment>
-
-
 ## Astro
 
+### `Uncaught ReferenceError: global is not defined`
```

Comment:
Tested that this is the specific error message you get in Astro (uses vite) when starting a new project